ydb/core/tablet/node_whiteboard.cpp

```diff
@@ -46,6 +46,10 @@ class TNodeWhiteboardService : public TActorBootstrapped<TNodeWhiteboardService>
         TabletIntrospectionData.Reset(NTracing::CreateTraceCollection(introspectionGroup));
 
         SystemStateInfo.SetHost(FQDNHostName());
+        const TString& slotName = AppData(ctx)->SlotName;
+        if (!slotName.Empty()) {
```

Comment:
Тогда уж и тут поправь :)
```suggestion 
        if (const TString& slotName = AppData(ctx)->SlotName; !slotName.Empty()) {
```